roles/openshift_logging/tasks/main.yaml

```diff
@@ -12,6 +12,14 @@
 
 - debug: msg="Created temp dir {{mktemp.stdout}}"
 
+- name: Create local temp directory for doing work in
```

Comment:
I did not find it though I thought we made that change LONG ago.  Please correct me if I am basing this off a stale commit.